src/OpenVocabularyAssociationStorage.php

```diff
@@ -78,6 +78,22 @@ public function loadAssociationsByField(string $field_id): array {
     return $this->loadMultiple($results);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function loadAssociationsByVocabulary(string $vocabulary_id): array {
```

Comment:
You introduce this but how come you don't use it anywhere? I'd have imaged in OpenVocabularyDeleteForm::buildForm where you run the same query no?

Reply:
Oh I didn't push that it seems, true